hack/bin/buildah-make-images.sh

```diff
@@ -9,7 +9,7 @@ EXECUTABLES=${EXECUTABLES:-"cannon brig cargohold galley gundeck federator brig-
 CONTAINER_NAME="output"
 DOCKER_TAG=${DOCKER_TAG:-$USER}
 
-buildah containers | awk '{print $5}' | grep "$CONTAINER_NAME" || \
+buildah containers | awk '{print $5}' | grep "$CONTAINER_NAME" ||
```

Comment:
sorry @akshaymankar , my bash auto-correcting/formatting toolchain (https://github.com/z0mbix/vim-shfmt) removes the backslash here on file save.